### DISCUSSION TITLE
Doubt on finegrain/finegrain-box-segmenter

### DISCUSSION THREAD

**Discussion by rishabh063** [finegrain-ai/refiners#435] [Q&A] [answered]
I was looking at its implementation and feels like you guys are just cropping and passing the cropped image to mavenet ? or am i missing something ?

**Comment by piercus** [comment, marked as answer]
@rishabh063 Thanks for reaching out! 

In this v0.1, box prompting is not baked in yet, but our training is tailored for this naive box prompting (unlike the official MVANet weights). 

We’re working on an “integrated” approach that will further boost accuracy - stay tuned!

**Reply by rishabh063** [reply]
Oh  , can you guys share the training script or some other info on how you are planning on doing it ?

**Reply by piercus** [reply]
We don't know yet if we'll open source the training script, but we already shared key aspects in the [release note](https://huggingface.co/finegrain/finegrain-box-segmenter#training), like:
> We moved away from the usual random crop approach. Instead, we designed our custom cropping strategy to make sure the model understands what object to select in a given bounding box. We used batch sizes of 5 to improve the training stability

**Reply by rishabh063** [reply]
couldn't make sense out of , any more info can help 

btw thanks for open sourcing it
